lib/airbrake-ruby.rb

```diff
@@ -84,6 +84,12 @@ module Airbrake
   #  special cases where we need to work around older implementations
   JRUBY = (RUBY_ENGINE == 'jruby')
 
+  # @return [Boolean] true if this Ruby supports ... argument forwarding.
+  HAS_ARGUMENT_FORWARDING = (
```

Comment:
This constant now looks exactly the same as `Airbrake::HAS_SAFE_LEVEL`.
We could give it another name, so it's more general and we could use it everywhere but I struggle to suggest anything decent. 🤷‍♂️

Reply:
HAS_SAFE_LEVEL is true if MRI and version < 2.7, and probably never any other Ruby platforms.

HAS_ARGUMENT_FORWARDING is true for MRI and >= 2.7, and eventually might be true for JRuby and TruffleRuby once they start doing 2.7+ compatibility.

Reply:
I had trouble coming up with a name too. HAS_ARGUMENT_FORWARDING is at least topical; it refers to the `def a(...); b(...); end` argument forwarding syntax.

Reply:
Oh yeah, my bad for not noticing they are different. Let's just disregard this comment then.

As for TruffleRuby, we might not even support it. I've never tried running tests there.

`HAS_ARGUMENT_FORWARDING` sounds like a good name to me, and it's easy to understand.